lib/add_epoch_column.py

```diff
@@ -0,0 +1,21 @@
+from pyspark.sql.functions import col, when
+from pyspark.sql import DataFrame
+
+def add_epoch_column(df: DataFrame, periods = [1901,1918,1926,1939,1954,1970,1985,1994,2009]) -> DataFrame:
+
+  assert df.filter(df.rok_wydania_produkcji == "\\N").count() == 0
+  assert dict(df.dtypes)["rok_wydania_produkcji"] == "int"
+
+  df_periods = df.withColumn('period',
+                             when(col('rok_wydania_produkcji') <= periods[0], "1")
```

Comment:
dla komputera i modelu to wszystko jedno czy liczymy od zera czy od jedynki, ale ta niezgodność wartości, może być myląca. Pozostawiam decyzję czy to wdrożyć.

Reply:
kod będzie wyglądać inaczej, jeśli formuła z reduce'm zadziała. Jeżeli powyższa forma funkcji zostanie, to myślę, żeby ją zostawić w takim stanie.